features/car/access.feature

```diff
@@ -220,3 +220,15 @@ Feature: Car - Restricted access
         Then routability should be
             | highway | toll        | bothw |
             | primary | yes         |       |
+
+
+     Scenario: Car - barrier=gate should be routed over unless explicitely forbidden
+        Then routability should be
+            | node/barrier | access     | bothw |
+            | gate         |            | x     |
```

Comment:
So shouldn't it be the other way around? If you want to add the test, it should be as a `todo` and have the correct default value. Otherwise it might imply to someone that we want it to be this way.